presto-docs/src/main/sphinx/functions/map.rst

```diff
@@ -111,7 +111,7 @@ Map Functions
 
 .. function:: map_top_n_keys(x(K,V), n) -> array(K)
 
-    Returns top ``n`` keys in the map ``x`` by sorting its keys in descending order.
+    Returns top ``n`` keys in the map ``x`` by sorting its keys in descending order. Keys are used to break ties with the max key being chosen. Both keys and values should be orderable.
```

Comment:
I think you updated the wrong documentation.  This should be at map_top_n (a few entries below), not map_top_n_keys

Reply:
Apologies and Thank you @rschlussel . Fixed.